pkg/operator/ceph/webhook.go

```diff
@@ -63,6 +68,151 @@ func isSecretPresent(ctx context.Context, context *clusterd.Context) (bool, erro
 	return true, nil
 }
 
+func webhookcreate(contextContext *context.Context, clusterdContext *clusterd.Context) {
```

Comment:
I could see the operator waiting for the cert mgr to be ready if it's detected to be installed, but would not be creating the resources related to the webhook.